instrumentation/github.com/syndtr/goleveldb/leveldb/splunkleveldb/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/signalfx/splunk-otel-go/instrumentation/github.com/syndtr/goleveldb/leveldb/splunkleveldb
 
-go 1.16
+go 1.15
```

Comment:
Please isolate changes to the new splunkbuntdb package. If we want to make this change lets do so in a separate PR.